src/index.html

```diff
@@ -4,9 +4,409 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    crossorigin="anonimus"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img src="images/logo.png" alt="logo">
+      </a> 
+      <nav>
+        <ul class="header__list">
+          <li class="header__list-item">
+            <a href="#" class="is-active header__list-item-name">
+              Apple
+            </a>
+          </li>
+
+          <li class="header__list-item">
+            <a href="#" class="header__list-item-name">Samsung</a>
+          </li>
+
+          <li class="header__list-item">
+            <a href="#" class="header__list-item-name">Smartphones</a>
+          </li>
+
+          <li class="header__list-item">
+            <a 
+              href="#"
+              data-qa="nav-hover"
+              class="header__list-item-name"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__list-item">
+            <a href="#" class="header__list-item-name">Gadgets</a>
+          </li>
+
+          <li class="header__list-item">
+            <a href="#" class="header__list-item-name">Tablets</a>
+          </li>
+
+          <li class="header__list-item">
+            <a href="#" class="header__list-item-name">Photo</a>
+          </li>
+
+          <li class="header__list-item">
+            <a href="#" class="header__list-item-name">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <div class="catalog__block">
+          <div class="catalog__card card" data-qa="card">
+            <img 
+              class="card__product-photo"
+              src="images/imac.jpeg"
+              alt="photo"
+            >
+            <p class="card__product-name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+            <p class="card__product-code">
+              Product code: 195434
+            </p>
```

Comment:
```suggestion
          <div class="catalog__card card" data-qa="card">
            <img 
              class="card__product-photo"
              src="images/imac.jpeg"
              alt="photo"
            >
            
            <p class="card__product-name">
              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
            </p>
            
            <p class="card__product-code">
              Product code: 195434
            </p>
```
Correct your code style